tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java

```diff
@@ -51,11 +51,15 @@
  */
 public class CDIPlainInjectionTest extends Arquillian {
 
+    private static final String DEFAULT_PROPERTY_BEAN_KEY = "org.eclipse.microprofile.config.tck.CDIPlainInjectionTest.defaultPropertyBean.configProperty";
```

Comment:
OK, I see. It's very confusing in this case, but I guess we can't do anything about it since it's already in the final version. I'm fine with that since it's a rare edge case to have a nested CDI bean class.